content/stats/view.md

```diff
@@ -57,7 +57,7 @@ func enableViews() error {
 
 	lineCountView = &view.View{
 		Name:        "myapp/lines_in",
-		Measure:     mLatencyMs,
```

Comment:
Please revert this change, we are applying a count aggregation on mLatencyMs which is invoked once for every line unless am mistaken. If mLatencyMs is used more than once, then we can just use mBytes*